### PR TITLE
Misc fixes (unimportant)

### PR DESCRIPTION
**Pull request by AlexKVal**
While I was following and reading through the tutorial of the `react_on_rails` gem,
I've noticed these missed bits.
Old links, old names etc.
Though nothing of importance.
🍒

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/514)

### CODE REVIEW

**Comment by justin808** [comment]
Good changes! See my comments.

Thanks!

---

Reviewed 6 of 6 files at r1.
Review status: all files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---

*[lib/generators/USAGE, line 24 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/514#-KOq_3-iATtsddiDUvEk:-KOq_3-iATtsddiDUvEl:buadgo1) ([raw file](https://github.com/shakacode/react_on_rails/blob/8af18dec7d6fd8dfa1449e59945a9413975cdc23/lib/generators/USAGE#L24)):*
> ```
> More Details:
> 
>     `https://github.com/shakacode/react_on_rails/tree/master/docs`
> ```

https://github.com/shakacode/react_on_rails/blob/master/docs/basics/generator.md is the correct link

---

*[lib/generators/react_on_rails/base_generator.rb, line 26 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/514#-KOq_9i0YSVRWfHJWBEB:-KOq_9i0YSVRWfHJWBEC:b92sf4m) ([raw file](https://github.com/shakacode/react_on_rails/blob/8af18dec7d6fd8dfa1449e59945a9413975cdc23/lib/generators/react_on_rails/base_generator.rb#L26)):*
> ```Ruby
>         data = <<-DATA.strip_heredoc
>           # React on Rails
>           npm-debug.log.*
> ```

do we always get a `.<number>`? this is a glob, not a regexp

---

*[spec/dummy/client/webpack.client.base.config.js, line 27 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/514#-KOq_Nb0iljj2mzcB3fO:-KOq_Nb0iljj2mzcB3fP:b-8fx2u7) ([raw file](https://github.com/shakacode/react_on_rails/blob/8af18dec7d6fd8dfa1449e59945a9413975cdc23/spec/dummy/client/webpack.client.base.config.js#L27)):*
> ```JavaScript
> 
>     // This will contain the app entry points defined by
>     // webpack.client.rails.hot.config and webpack.client.rails.build.config
> ```

:+1: 

---

*[spec/dummy/client/webpack.client.base.config.js, line 29 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/514#-KOq_QMcI-5fz3SWzMQC:-KOq_QMcI-5fz3SWzMQD:bjofuc0) ([raw file](https://github.com/shakacode/react_on_rails/blob/8af18dec7d6fd8dfa1449e59945a9413975cdc23/spec/dummy/client/webpack.client.base.config.js#L29)):*
> ```JavaScript
>     // webpack.client.rails.hot.config and webpack.client.rails.build.config
>     app: [
>       // './app/assets/styles/global-styles.scss',
> ```

should we remove this comment?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/react_on_rails/514#-:-KOq_Z0JxukUwUAMDUm9:b-rgabr3)*

**Comment by AlexKVal** [comment]
Review status: all files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---

*[lib/generators/USAGE, line 24 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/514#-KOq_3-iATtsddiDUvEk:-KOsiuQoWCnW5fOQHjJ3:b-wkz09k) ([raw file](https://github.com/shakacode/react_on_rails/blob/8af18dec7d6fd8dfa1449e59945a9413975cdc23/lib/generators/USAGE#L24)):*
<details><summary><i>Previously, justin808 (Justin Gordon) wrote…</i></summary>
> https://github.com/shakacode/react_on_rails/blob/master/docs/basics/generator.md is the correct link
</details>
Thank you! I have not read all files in the `docs` folder.

---

*[lib/generators/react_on_rails/base_generator.rb, line 26 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/514#-KOq_9i0YSVRWfHJWBEB:-KOsjAP7FOTpuSZ1BY2H:b-oxsk2i) ([raw file](https://github.com/shakacode/react_on_rails/blob/8af18dec7d6fd8dfa1449e59945a9413975cdc23/lib/generators/react_on_rails/base_generator.rb#L26)):*
<details><summary><i>Previously, justin808 (Justin Gordon) wrote…</i></summary>
> do we always get a `.<number>`? this is a glob, not a regexp
</details>
So lame of me. I must have been sleeping already at the time :)

---

*[spec/dummy/client/webpack.client.base.config.js, line 27 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/514#-KOq_Nb0iljj2mzcB3fO:-KOsjPOBYyDePoGnufOW:b4tmoq5) ([raw file](https://github.com/shakacode/react_on_rails/blob/8af18dec7d6fd8dfa1449e59945a9413975cdc23/spec/dummy/client/webpack.client.base.config.js#L27)):*
<details><summary><i>Previously, justin808 (Justin Gordon) wrote…</i></summary>
> :+1: 
</details>
I would remove at all these comments in `webpack.*.config` files, but maybe they will be in help for newcomers.

---

*[spec/dummy/client/webpack.client.base.config.js, line 29 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/514#-KOq_QMcI-5fz3SWzMQC:-KOskIh_w4AmMiEVRLD6:bgy54kq) ([raw file](https://github.com/shakacode/react_on_rails/blob/8af18dec7d6fd8dfa1449e59945a9413975cdc23/spec/dummy/client/webpack.client.base.config.js#L29)):*
<details><summary><i>Previously, justin808 (Justin Gordon) wrote…</i></summary>
> should we remove this comment?
</details>
There are many such dead code lines throughout the project.
I wasn't sure about your stance about them.
Next time I'll propose to remove them too.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/react_on_rails/514)*

**Comment by AlexKVal** [comment]
I've fixed my "fixes" 😄

**Comment by justin808** [comment]
:lgtm:

---

Reviewed 6 of 6 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/react_on_rails/514#-:-KOsn_xFT-0oVCV_ps9p:bnfp4nl)*

File: .gitignore [hunk 1]
```diff
@@ -25,7 +25,7 @@ node_modules
 /node_package/lib
 
 npm-debug.*
-spec/dummy/client/npm-debug.log.*
+spec/dummy/client/npm-debug.log*
```
Comment (justin808) [inline .gitignore:28]: FWIW, @AlexKVal there's no reason this shouldn't be in your global git ignore!

Comment (AlexKVal) [inline .gitignore:28]: ## Totally agree. Thank you. 🍒 

https://github.com/AlexKVal/dotfiles/commit/fbea3f645dd

**Pull request merged**